config/app.yaml

```diff
@@ -0,0 +1,13 @@
+env :
+  production :
+  staging :
+  development :
+    db :
+#      net  : "tcp"        # tcp|sock
+#      host : "127.0.0.1"  # not required for net:sock
+#      port : 3306
+      name : "myBusiness"
+      user : "root"
+      pass : "rootpassword"
```

Comment:
I think I'll remove this line entirely and we can look at enforcement inline in the app itself